exporter/pagerdutyexporter/README.md

```diff
@@ -0,0 +1,53 @@
+# PD Exporter
+
+Exports data to a PD back-end.
+By default, this exporter requires TLS and offers queued retry capabilities.
+
+Supported pipeline types: traces
+
+## Getting Started
+
+The following settings are required:
+
+- `endpoint` (no default): URL to which the exporter is going to send PD trace data.
+<!-- - `format` (default = `JSON`): The format to sent events in. Can be set to `JSON` or `proto`. -->
+
+By default, TLS is enabled and must be configured under `tls:`:
+
+- `insecure` (default = `false`): whether to enable client transport security for
+  the exporter's connection.
+
+As a result, the following parameters are also required under `tls:`:
+
+- `cert_file` (no default): path to the TLS cert to use for TLS required connections. Should
+  only be used if `insecure` is set to false.
+- `key_file` (no default): path to the TLS key to use for TLS required connections. Should
+  only be used if `insecure` is set to false.
+
+The following settings are optional:
+
+- `defaultservicename` (default = `<missing service name>`): What to name
```

Comment:
I suggest adding some instructions of the expected format otherwise, I imagine users may confuse this and wrap the service name in angled brackets.